src/components/AnnotationOverlay/AnnotationOverlay.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+
+import core from 'core';
+import selectors from 'selectors';
+
+import './AnnotationOverlay.scss';
+
+const MAX_WORDS = 100;
+
+const AnnotationOverlay = () => {
+  const isDisabled = useSelector(state => 
+    selectors.isElementDisabled(state, 'annotationOverlay')
+  );
+  const [isOpen, setIsOpen] = useState(false);
+  const [annotation, setAnnotation] = useState();
+  const [repliesCount, setRepliesCount] = useState(0);
+  const [overlayPosition, setOverlayPosition] = useState({
+    left: 0,
+    top: 0
+  });
+  
+  const am = core.getAnnotationManager();
+  
+  useEffect(() => {
+    window.docViewer.on('mouseMove.hover', onMouseHover);
+    return () => {
+      window.docViewer.off('mouseMove.hover', onMouseHover);
```

Comment:
I don't think we need to use event namespace here. Doing `mouseMove` should be OK. Can we also use `core.addEventListener` just for consistency?

Reply:
Sure, sounds good!